scrapyd/app.py

```diff
@@ -23,7 +24,11 @@ def application(config):
     bind_address = config.get('bind_address', '127.0.0.1')
     poll_interval = config.getfloat('poll_interval', 5)
 
-    poller = QueuePoller(config)
+    if config.getboolean('fix_poll_order', False):
+        log.msg("Activating contrib modules for fix_poll_order")
+        poller = FixQueuePoller(config)
+    else:
+        poller = QueuePoller(config)
```

Comment:
Should do something similar to Launcher, etc. so that users can select any module (including their own), not just between these two modules.